test/spec/browserStorage.js

```diff
@@ -152,6 +152,18 @@ describe('browserStorage', () => {
         sameSite: 'strictly fakey'
       });
     });
+
+    it('setItem: will call storage.set, passing sessionCookie, secure and sameSite options ', () => {
```

Comment:
Is this the test that should have the title "but no sesssionCookie option"? It appears to be testing that the sessionCookie option is NOT passed through